rakelib/compile.rake

```diff
@@ -37,10 +37,8 @@ namespace "compile" do
   end
 
   task "logstash-core-java" do
-    unless File.exists?(File.join("logstash-core", "lib", "jars", "logstash-core.jar"))
-      puts("Building logstash-core using gradle")
-      safe_system("./gradlew", "assemble")
-    end
+    puts("Building logstash-core using gradle")
+    safe_system("./gradlew", "assemble")
```

Comment:
fyi: there's a rake helper `sh` which does raise on non-0 exit + will print the command run as well as output
```suggestion
    sh('./gradlew assemble')
```
`safe_system` is simply reinventing the wheel here